pkgs/development/python-modules/dbx/default.nix

```
@@ -0,0 +1,79 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, databricks-cli
+, scipy
+, pathpy
+, pathspec
+, pydantic
+, protobuf
+, tqdm
+, mlflow
+, azure-identity
+, ruamel_yaml
```

```suggestion
, ruamel-yaml
```
you are using an alias.